core/templates/pages/contributor-dashboard-page/services/contribution-and-review-backend-api.service.ts

```diff
@@ -113,7 +114,7 @@ export class ContributionAndReviewBackendApiService {
     }
     if (fetchType === this.REVIEWABLE_QUESTION_SUGGESTIONS) {
       return this.fetchReviewableSuggestionsAsync(
-        'skill', 'add_question', limit || 0, offset, sortKey);
+        'skill', 'add_question', limit || 0, offset, sortKey, null, topicName);
```

Comment:
Please always pass topic IDs and not topic names. The latter isn't a stable identifier.

Reply:
Hmm, the problem is that we don't have good support of topicIDs on this page. If we want topicID, we need to rework the translation-topic-selector [component](https://github.com/oppia/oppia/blob/5b0c07043f74c2ccef56309a4db49c82763f098b/core/templates/pages/contributor-dashboard-page/translation-topic-selector/translation-topic-selector.component.html#L4), and the translation-topic [service](https://github.com/oppia/oppia/blob/5b0c07043f74c2ccef56309a4db49c82763f098b/core/templates/pages/exploration-editor-page/translation-tab/services/translation-topic.service.ts#L4), because they only operate with the names. Also this selector and the topicName are currently [used](https://github.com/oppia/oppia/blob/5b0c07043f74c2ccef56309a4db49c82763f098b/core/templates/pages/contributor-dashboard-page/translation-opportunities/translation-opportunities.component.ts#L146) on the translation opportunities page. Not only that, we currently [get](https://github.com/oppia/oppia/blob/5b0c07043f74c2ccef56309a4db49c82763f098b/core/templates/pages/contributor-dashboard-page/contributor-dashboard-page.component.ts#L196) only topic names from the backend for the selector. So it would require a quite big refactoring. I honestly doubt I will be able to do all this, the code in this place confuses me a lot. 

Do you think it's better not to add it at all or add it with the topicName?

Reply:
Ah I see, thanks. In that case I think we'll need to stick with topicName for now, but I'd suggest filing an issue to use topicId instead since it's a more stable identifier. Thanks!

Reply:
SG, created an [issue](https://github.com/oppia/oppia/issues/19199), thanks!